assets/js/done-task.js

```diff
@@ -0,0 +1,28 @@
+import { CLICK_EVENT, HIDE_CLASS } from "/assets/js/constants.js";
+import createButton from "/assets/js/factory/createButton.js";
+
+function doneTaskEvent(done, edit, inputElement, tools, allTasks, id) {
```

Comment:
see https://github.com/rsa-wtag/Todo-App-with-JS/pull/2#discussion_r1161260192

```suggestion
function onTaskComplete(done, edit, inputElement, tools, allTasks, id) {
```